USER-INTERFACE-INTERFACE.md

```diff
@@ -740,10 +742,10 @@ With `customQueryOpt`, the limiting age and balance ranges depend on the user's
 records being returned. The results are always ordered by balance in this mode, and it is the case here too, that
 an empty array is handed back if no records can be retrieved. 
 
-Because of a cascade of optional values going along with this letter kind of a query, and also because it's a valid
-option to request just a single table view, null should be anticipated in various places, either at the position of
-individual tables (`payableOpt`, `receivableOpt`) or instead of the whole thing (`customQueryOpt`), which points to
-that no query of that kind was actually made in the first place. 
+Since this command as a whole is optional, and it is also a valid option to request just a single table view, instead of
```

Comment:
"This command as a whole is optional, and it is also a valid option to request just a single table view. Therefore [, of necessity,] null should be anticipated..."